packages/darklang/cli/bash.dark

```diff
@@ -0,0 +1,85 @@
+module Darklang =
+  module Cli =
+    module Bash =
+      /// Read the content of .bashrc file
+      let readBashrc () : Stdlib.Result.Result<String, String> =
+        let zhrcContent = Builtin.fileRead ".bashrc"
+
+        match zhrcContent with
+        | Ok content ->
+          content
+          |> Stdlib.String.fromBytesWithReplacement
+          |> Stdlib.Result.Result.Ok
+        | Error e -> Stdlib.Result.Result.Error e
+
+
+      /// Write content to .bashrc file
+      let writeToBashrc (content: String) : Stdlib.Result.Result<String, String> =
+        let writeResult = Builtin.fileWrite (Stdlib.String.toBytes content) ".bashrc"
+
+        match writeResult with
+        | Ok _ -> Stdlib.Result.Result.Ok "Wrote to .bashrc"
+        | Error e -> Stdlib.Result.Result.Error e
+
+
+      /// Delete an alias from .bashrc
+      let deleteAlias (alias: String) : Stdlib.Result.Result<String, String> =
+
+        let deleteAlias =
+          Stdlib.Process.runCommand $"sed -i '' '/alias {alias}=/d' ~/.bashrc"
+
+        if deleteAlias.exitCode == 0L then
+          Stdlib.Result.Result.Ok "Alias deleted"
+        else
+          Stdlib.Result.Result.Error deleteAlias.stderr
+
+
+      /// Add an alias to .bashrc
+      let addAlias
+        (alias: String)
+        (command: String)
+        : Stdlib.Result.Result<String, String> =
+        let aliasDefinition = $"alias {alias}='{command}'"
+
+        match (readBashrc ()) with
+        | Ok content ->
+          let newContent = content ++ "\n" ++ aliasDefinition
+          writeToBashrc newContent
+
+          Builtin.printLine
+            "Changes have been made to your .bashrc file. To apply these changes to your current terminal session, please run the following command:\nsource ~/.bashrc\n\n "
```

Comment:
Added this message because the apply `applyBashConfigChanges ()` function doesn't seem to work in bash (It works in zsh). The solution is either running  `source ~/.bashrc` manually or the user need to have this line `[[ -f ~/.bashrc ]] && . ~/.bashrc` added to there `bash_profile` file 
any thoughts on how to handle this?